circuit-runtime/src/commonMain/kotlin/com/slack/circuit/runtime/Navigator.kt

```diff
@@ -12,6 +12,8 @@ public interface Navigator {
 
   public fun pop(): Screen?
 
+  public fun peek(): Screen?
```

Comment:
Can you add a short doc? I know the other functions are missing docs but we wanna start being better about this :)

Reply:
Sure, attached in dfcb9e2e3e83c49890c081e07ba98e98c00879b4